src/classes/HH_Container_LCTRL.cls

```diff
@@ -564,7 +564,7 @@ public with sharing class HH_Container_LCTRL {
                 
                 for (String strF : setFields) {
                     // check field level security
-                    DescribeFieldResult dfr = UTIL_Describe.getFieldDescribe(UTIL_Namespace.StrTokenNSPrefix('Address__c'), strF);
```

Comment:
not sure why this is showing in your pull request since this already went into dev.  wonder if your branch is not up to date?